src/main/java/com/yahoo/bullet/parsing/Query.java

```diff
@@ -27,9 +29,9 @@
 @Getter @Setter @Slf4j
 public class Query implements Configurable, Initializable {
     @Expose
-    private Projection projection;
+    private Map<String, LazyExpression> projection;
     @Expose
-    private List<Clause> filters;
+    private LazyExpression filter;
```

Comment:
Can we force this to be a LazyBinary?

Reply:
Sure, if you can you make sure we can only put expressions that produce booleans in here without force casting it?